CWE/decisions/italy.txt

```diff
@@ -0,0 +1,95 @@
+political_decisions = {
```

Comment:
Please set the pre 1949 decisions as already fired in the history entry of Italy, under the 1950 bracket. See USA’s history file for an example.